colossalai/kernel/cuda_native/csrc/attention_infer_kernels/flash_attn/src/block_info.h

```diff
@@ -0,0 +1,41 @@
+/******************************************************************************
```

Comment:
Are these files the same as origin flash attn?

Reply:
not exactly same, I only kept forward codes over here.

Reply:
> Are these files the same as origin flash attn?

or we can directly use third-party (flash-attention-2) as our submodules?

Reply:
I think can we directly import and use it?

Reply:
OK!